examples/pytorch_sample/pytorch_smoke.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import os
+import json
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+
+from math import ceil
+from random import Random
+from torch.autograd import Variable
+from torchvision import datasets, transforms
+
+def run():
+    """ Simple Send/Recv for testing Master <--> Workers communication """
+    rank = dist.get_rank()
+    size = dist.get_world_size()
+    inp = torch.randn(2,2)
+    result = torch.zeros(2,2)
+    if rank == 0:
+        # Send the input tensor to all workers
+        for i in range(1, size):
+            dist.send(tensor=inp, dst=i)
+            # Receive the result tensor from all workers
+            dist.recv(tensor=result, src=i)
+            logging.info("Result from worker %d : %s", i, result)
+    else:
+        # Receive input tensor from master
+        dist.recv(tensor=inp, src=0)
+        # Elementwise tensor multiplication
+        result = torch.mul(inp,inp)
+        # Send the result tensor back to master
+        dist.send(tensor=result, dst=0)
+
+def init_processes(fn, backend='tcp'):
+    """ Initialize the distributed environment. """
+    dist.init_process_group(backend)
+    fn()
+
+def main():
+    logging.info("Torch version: %s", torch.__version__)
+    
+    config_json = os.environ.get("TF_CONFIG", "{}")
```

Comment:
Don't need TF_CONFIG in here. We should probably remove it from the operator anyways